<style>
    @media (max-width:450px){
        #table2 thead tr th:nth-child(1){padding:15px 5px 15px 15px;}
        #table2 thead tr th:nth-child(3){padding:15px 5px;}
        #table2 tbody tr td:nth-child(1){padding:15px 5px 15px 15px;}
        #table2 tbody tr td:nth-child(2){padding:15px 5px;}
        #table2 tbody tr td:nth-child(3){padding:15px 5px;}
    }
</style>
<h1 id="abm-category-home-title">CRISPR Stable Knockout Cell Line Generation</h1>
<nav id="abm-navbar" class="navbar navbar-fixed-top abm-navbar abm-affix abm-category-navbar-relative hidden-xs">
	<div class="abm-container-fluid">
		<div id="abmNavbar" class="collapse navbar-collapse abm-navbar-collapse">
			<ul class="nav navbar-nav navbar-6">
				<li><a href="#abm-category-section1">Overview</a></li>
				<li><a href="#abm-category-section2">Service Details</a></li>
				<li><a href="#abm-category-section3">Additional Info</a></li>
				<li><a href="#abm-category-section4">Related Products</a></li>
				<li><a href="#abm-category-section5">Documents &amp; FAQS</a></li>
				<li><a href="#abm-category-section7">Citations</a></li>
			</ul>
		</div>
	</div>
</nav>
<div id="abm-category-section1" class="abm-container-fluid">
	<div class="row">
		<div class="col-sm-8 col-xs-12">
			<p>Trust <strong>abm</strong> to knock out your gene of interest in your chosen Human, Mouse, or Rat cell line using our CRISPR sgRNA lentiviral system. We have successfully completed many knockout projects in a variety of cell lines, including fibroblasts, blood cells, ovarian cells, hepatocytes, and many more. Just tell our dedicated and highly experienced team which gene to target, how your cells grow, and we'll do the sgRNA design, cloning, lentiviral packaging, and stable cell line generation for you! <br /><br /> For your experiment to evaluate the effects of the knockout, be sure to ask us to make a Wild Type Control Cell Line Expressing Cas9 for comparison in the same cellular lineage. <br /><br /> Interested in a <strong>100% Guaranteed Knockout service</strong>? Find out more <a class="orange-link" href="/contract-research-cro-services/crispr-genome-editing-services/guaranteed-crispr-ko-cell-line-generation">here</a>.</p>
			<br />
			<p><a href="/crispr-stable-knockout-cell-line-generation-inquiry" target="_blank"><button class="custom-service-inquire-button-normal" style="float: left;">Inquire</button></a></p>
			<br /> <br /> <b class="subtitle">Workflow</b>
			<div>
				<p class="banners abm-category-workflow"><a href="javascript:void(0)"><img src="/pub/media/catalog/customservice/crispr-stable-knockout-cell-line/CRISPR-workflow.png" alt="Workflow" /></a></p>
			</div>
		</div>
		<div class="col-sm-4  col-xs-12">
			<div class="mini-box2">
				<div class="banners">
					<div><a href="https://www.abmgood.com/marketing/knowledge_base/CRISPR_Cas9_Introduction_Part1.php"> <img src="/pub/media/catalog/customservice/custom_crispr_sgrna_lentiviral_vectors_&amp;_viruses/Resources_Introduction-to-CRISPR.png" alt="Introduction to CRISPR" width="100%" /> <br /> </a> <strong>Introduction to CRISPR - Knowledge Base</strong></div>
				</div>
				<br />
				<p class="bluetext">"We are quite happy with the knockout cells that you made. This was a tricky knockout and hemizygous removal was much more prevalent than the double knockout, but you did it. In addition to your excellent genomic sequence characterization of both mutant LIF alleles, I have confirmed in our lab that the tumor cells have lost all mouse LIF production."</p>
				<i>Dr. Robert Jackman, Boston University, Mouse LIF CRISPR Knock Out C26 Tumor Cell Line Generation and Screening Service, View case study <a class="orange-link" href="/pub/media/catalog/customservice/crispr-stable-knockout-cell-line/CRISPR-Stable-KO-Service-Case-Study.pdf">here</a>.</i>
			</div>
		</div>
	</div>
</div>
<div id="abm-category-section2">
	<div class="abm-container-fluid">
		<div class="row">
			<div class="col-lg-8">
				<h3 class="abm-perfect-h3">Service Details</h3>
				<table id="table1" class="bootstrap-table bootstrap-table-hover abm-service-table abm-perfect-table2">
					<thead>
						<tr>
							<th>Our Basic Package (C208) Includes</th>
							<th>Deliverables</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<td>
								<ol style="list-style-type: decimal; padding-left: 0px;">
									<li>Cell line Validation (e.g. infection condition optimization, drug selection evaluation, Single cell clonability evaluation etc.)</li>
									<li>sgRNA vector design, construction &amp; delivery into cells</li>
									<li>Clonal screening and selection</li>
									<li>Screening &amp; validation of knockout by Sanger sequencing**</li>
									<li>Expansion &amp; cryopreservation of selected clone(s)</li>
									<li>Microbial/sterility testing</li>
								</ol>
							</td>
							<td style="text-align: left; color: black;">Sequence verified knockouts (at least 1 clone, 2 vials per clone)**, microbial/sterility tested with a service report.</td>
						</tr>
					</tbody>
				</table>
				<table id="table2" class="bootstrap-table bootstrap-table-hover abm-service-table abm-perfect-table1">
					<thead>
						<tr>
							<th>Add&#8209;On&nbsp;Services</th>
							<th>Unit</th>
							<th>CAT.&nbsp;NO.</th>
							<th>PRICE</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<td>WT Control Cell Line Expressing Cas9 for Comparison <i>(50% discount for academic customers if ordered with the basic package)</i></td>
							<td>1&nbsp;Cell&nbsp;Line</td>
							<td>C142</td>
							<td>$<span class="customServicePriceGeoIP">1590.00</span></td>
						</tr>
						<tr>
							<td>Additional Clones</td>
							<td>1&nbsp;Clone</td>
							<td>C143</td>
							<td>$<span class="customServicePriceGeoIP">250.00</span></td>
						</tr>
						<tr>
							<td>Additional Vials of Delivered Cells^^^</td>
							<td>1&nbsp;Vial</td>
							<td>C144</td>
							<td>$<span class="customServicePriceGeoIP">80.00</span></td>
						</tr>
						<tr>
							<td>Western Blot Validation Service (Up to 10 Clones Tested)<sup>##</sup></td>
							<td>1&nbsp;Service</td>
							<td>C145</td>
							<td>$<span class="customServicePriceGeoIP">975.00</span></td>
						</tr>
						<tr>
							<td>Off-Target Analysis by Whole Genome Sequencing</td>
							<td>Per&nbsp;Clone</td>
							<td>C146</td>
							<td>$<span class="customServicePriceGeoIP">4000.00</span></td>
						</tr>
						<tr>
							<td>Additional Rounds of Selection &amp; Screening by Sanger Sequencing</td>
							<td>1&nbsp;Screening</td>
							<td>C147</td>
							<td>$<span class="customServicePriceGeoIP">835.00</span></td>
						</tr>
						<tr>
							<td>CRISPR-Cas9 Targeted Amplicon Sequencing</td>
							<td>1&nbsp;Service</td>
							<td>IA00100</td>
							<td>$<span class="customServicePriceGeoIP">600.00</span></td>
						</tr>
						<tr>
							<td>Cell Line Authentication:<br />STR Profiling of 1 WT Parental Cell Line &amp; 1 Knock-Out Cell Line Clone<br /><i>(50% discount if ordered with the basic package)</i></td>
							<td>1&nbsp;Service</td>
							<td>C287</td>
							<td>$<span class="customServicePriceGeoIP">150.00</span></td>
						</tr>
					</tbody>
				</table>
				<p><a href="/crispr-stable-knockout-cell-line-generation-inquiry" target="_blank"><button class="custom-service-inquire-button-normal" style="float: right;">Inquire</button></a></p>
				<br /> <b class="subtitle" style="font-size: 1em;">Notes:</b><br />
				<div style="font-size: 13px; color: grey;">*A deposit is required to initiate all CRISPR Cell Line Projects<br /> **Choose from HEK293, HEK293T, A549, HeLa, MDCK, A375, HepG2, HT1080 or U87MG. These cell lines are available as Cas-9 expressing versions or parental versions. Cas9 expressing version will be used by default. If the parental version is preferred, please request prior to order placement.<br /> **The above pricing is valid for a single gene knockout for diploid loci only.<br /> #Customer is required to provide at least 2 million cells, 1L of propagation media/any special coated flasks if needed (if cells need DMEM or RPMI, abm will supply media). Also, the cell line must tolerate single cell cloning and display adequate transfection and transduction efficiency.<br /> ##Pre-validated antibody needs to be provided by customer with appropriate positive control (preferably with supporting data).<br /> ^All add on services need to be indicated prior to order placement as the project design will need to be finalized prior to start of the service.<br /> ^^ Final Lead time may vary depending on actual growth rate of cell line while expanding from single cell.<br /> Please note that if the gene to be knocked-out may be essential to cell survival, it is up to the end-user to proceed with the services. abm is unable to guarantee cell survival in these cases and will only attempt to rescue the clones under these conditions. abm is not accountable for cell survival if rescuing the clones (instructions to be provided customers) is unsuccessful.<br /> ^^^ Additional media or supplements may be required for requests of additional vials</div>
			</div>
			<div class="col-lg-4"></div>
		</div>
	</div>
</div>
<div id="abm-category-section3" class="abm-container-fluid">
	<div class="row">
		<div class="col-lg-8">
			<h3 class="abm-perfect-h3">Additional Info</h3>
			<div>
				<a class="collapsed" href="#collapseAOne" data-toggle="collapse" data-parent="#accordion"><b class="subtitle">CRISPR Stable KO Service Case Study</b><i class="fa fa-angle-down OrderAngleDown" style="padding-left: 0.4em; font-size: 3rem;"></i></a><br />
				<div id="collapseAOne" class="panel-collapse collapse" style="border-left: none;">
					<div>
						<strong>Using CRISPR to develop a biallelic LIF knockout in Mouse Colon Carcinoma Cells</strong>
						<ul style="list-style-type: disc; padding-left: 0px;">
							<li>LIF locus in a Mouse Colon Carcinoma Cell Line, was knocked out using CRISPR targeted genome editing.</li>
							<li>Surveyor assay and sequencing results showed genome editing.</li>
							<li>After monoclonal selection biallelic knockout was confirmed by sequencing.</li>
							<li><strong>abm</strong> will generate the desired stable cell line and perform QC.</li>
						</ul>
						<br /> <strong>Phase #1: Cas9 and sgRNA Delivery</strong>
						<ul style="list-style-type: disc; padding-left: 0px;">
							<li>Three sgRNA were designed against mouse LIF locus (Mus musculus, NM_008501). Software analysis was performed to ensure the sgRNA had no predicted off targets binding sites. The selected sgRNA design was then cloned into the pLenti-U6-sgRNA-SFFV-Cas9-2A-Puro All-in-One lentivector (Figure 1).</li>
							<li>Recombinant Lentiviruses were packaged using abmâ€™s second generation Lentiviral packaging system. A multiplicity of infection (MOI) of 5 was used to transduce the cells.</li>
						</ul>
						<br />
						<div>
							<p class="banners abm-category-workflow"><a href="javascript:void(0)"><img src="/pub/media/catalog/customservice/crispr-stable-knockout-cell-line/pLenti-U6-sgRNA-SFFV-Cas9-2A-Puro-NEW.png" alt="Workflow" /></a></p>
						</div>
						<i>Figure 1. pLenti-U6-sgRNA-SFFV-Cas9-2A-Puro lentivector is an all-in-one vector for co-expressing sgRNA and Cas9 in mammalian cells. Expression of sgRNA is driven by the U6 promoter, a strong constitutive Pol III promoter. An SFFV promoter drives expression of the Cas9-2A-Puro cassette. By using the Cas9-2A-Puro cassette, cells can be directly screened for expression of Cas9, as they will be resistant to Puromycin.</i><br /><br /> <strong>Phase #2: First Round of Colony Screening for Edited Clones</strong>
						<ul style="list-style-type: disc; padding-left: 0px;">
							<li>Cell colonies are isolated after puromycin selection. Genomic DNA was extracted and the surveyor assay was performed to confirm genomic editing of the LIF locus.</li>
							<li>A single band in a surveyor assay at the wild-type (WT) size indicates no editing has occurred; two smaller bands (that sum to the length of the WT) indicate editing has taken place.</li>
							<li>The surveyor assay (Figure 2) indicated that Colony 3 and 6 were edited; colony 2 was not edited; and colony 1 was inconclusive.</li>
						</ul>
						<br />
						<div>
							<p class="banners abm-category-workflow"><a href="javascript:void(0)"><img src="/pub/media/catalog/customservice/crispr-stable-knockout-cell-line/Surveyor.png" alt="Workflow" /></a></p>
						</div>
						<i>Figure 2. The surveyor assay indicated that Colony 3 and 6 were edited; colony 2 was not edited; and colony 1 was inconclusive.</i><br /><br /> <strong>Phase #3: Sequence Analysis of the Edited Colonies</strong>
						<ul style="list-style-type: disc; padding-left: 0px;">
							<li>PCR products from Colonies 3 and 6 were further analyzed via Sanger Sequencing to determine the nature of the knockout (Figure 3).</li>
							<li>For colony 3 only one mutant sequence was detected, indicating that these cells are likely only heterozyotic knockouts. In colony 6 two different mutant sequences were detected.</li>
						</ul>
						<br />
						<div>
							<p class="banners abm-category-workflow"><a href="javascript:void(0)"><img src="/pub/media/catalog/customservice/crispr-stable-knockout-cell-line/Sequence-Alignment 1.png" alt="Workflow" /></a></p>
						</div>
						<i>Figure 3. For colony 3 only one mutant sequence was detected, indicating that these cells are likely only heterozyotic knockouts. In colony 6 two different mutant sequences were detected.</i><br /><br /> <strong>Phase #4: Second Round of Selection for Monoclonal Biallelic Knockout Clones</strong>
						<ul style="list-style-type: disc; padding-left: 0px;">
							<li>Colony 6 was serial diluted into 96 well plates for monoclonal selection. Genomic DNA was extracted from these clones (i.e. 6a, 6b..), PCR amplified, cloned and sequenced.</li>
							<li>Of the colony 6 clones, sequencing showed that only clone 6a had a frameshift mutation in both alleles (Figure 4). A frameshift mutation disrupts the open reading frame, resulting in nonsense mediated decay of mRNA transcript.</li>
						</ul>
						<br />
						<div>
							<p class="banners abm-category-workflow"><a href="javascript:void(0)"><img src="/pub/media/catalog/customservice/crispr-stable-knockout-cell-line/Sequence-Alignment 2.png" alt="Workflow" /></a></p>
						</div>
						<i>Figure 4. Clones 6a, 6b and 6d all showed biallelic editing. Only clone 6a had frame shift mutations in both alleles. No WT sequences were detected in all subclones.</i><br />
						<ul style="list-style-type: disc; padding-left: 0px;">
							<li>Further sequencing of clone 6a confirmed that only two mutant alleles were present, the 2 bp and 4 bp deletions, and that no WT or other mutations were detected (Figure 5).</li>
						</ul>
						<div>
							<p class="banners abm-category-workflow"><a href="javascript:void(0)"><img src="/pub/media/catalog/customservice/crispr-stable-knockout-cell-line/Sequence-Alignment 3.png" alt="Workflow" /></a></p>
						</div>
						<i>Figure 5. Further sequencing of 6a confirmed biallelic knock-out. No WT sequences were detected.</i><br /><br /> <strong>Phase #5: Confirmation of Knockout by Next Generation Amplicon Sequencing</strong>
						<ul style="list-style-type: disc; padding-left: 0px;">
							<li>With next generation sequencing hundreds of thousands of alleles can be sequenced at once, resulting in a more robust dataset. By contrast Sanger sequencing is only feasible for 1-100 clones and therefore it can miss a large proportion of the population.</li>
							<li>Next generation sequencing was performed at each stage of selection to evaluate knockout (Figure 6). Before editing, only WT sequences were observed. After the first round of selection colony 6 showed a mixture of edited (70%) and WT (30%) sequences. Finally after monoclonal selection, clone 6a showed only edited sequences with no WT alleles present.</li>
						</ul>
						<br />
						<div>
							<p class="banners abm-category-workflow"><a href="javascript:void(0)"><img src="/pub/media/catalog/customservice/crispr-stable-knockout-cell-line/NGS-KO.png" alt="Workflow" /></a></p>
						</div>
						<i>Figure 6. Next Generation Sequencing for CRISPR Knockout screening. A) Before knockout only WT sequences are detected. B) After Cas9 and sgRNA delivery, the first round of selection shows a mixed distribution of indel and WT sequences. C) After the second round of selection only knockouts remain.</i>
					</div>
				</div>
			</div>
			<div>
				<a class="collapsed" href="#collapseATwo" data-toggle="collapse" data-parent="#accordion"><b class="subtitle">Additional Information &amp; MTA</b><i class="fa fa-angle-down OrderAngleDown" style="padding-left: 0.4em; font-size: 3rem;"></i></a><br />
				<div id="collapseATwo" class="panel-collapse collapse" style="border-left: none;">
					<div>Orders of this service are subjected to the completion of a signed Material Transfer Agreement (MTA) by the purchasing individual/institution for each order. If you have any questions regarding this, please contact us at technical@abmgood.com. The end user acknowledges that the Materials provided under abm's MTA does not grant a license for commercial use or imply any ownership rights or any intellectual property rights relating to the Materials.</div>
				</div>
			</div>
		</div>
		<div class="col-lg-4"></div>
	</div>
</div>
<div id="abm-category-section4" class="abm-container-fluid">
	<div class="row">
		<div class="col-lg-8">
			<h3 class="abm-perfect-h3">Related Products</h3>
			<div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
				<div class="row">
					<div class="module">
						<ul class="htmlcontent-home">
							<li class="col-xs-6 col-sm-6 col-md-6">
								<div class="banners">
									<div class="serviceimages" style="text-align: left;">
										<a href="/CRISPR-Guaranteed-KO-Cell-Line-Generation.html"> <img src="/pub/media/catalog/category/imgs/perfect_page/Related-Products-tube.png" /> <br /> </a>
										<div class="abm-custom-service-image-title"><a class="black-link" href="/CRISPR-Guaranteed-KO-Cell-Line-Generation.html">Guaranteed CRISPR Knockout Cell Line Generation Service</a></div>
									</div>
								</div>
							</li>
							<li class="col-xs-6 col-sm-6 col-md-6">
								<div class="banners">
									<div class="serviceimages" style="text-align: left;">
										<a href="/CRISPR-Stable-Knockin-Cell-Line-Generation.html"> <img src="/pub/media/catalog/category/imgs/perfect_page/Related-Products-tube.png" /> <br /> </a>
										<div class="abm-custom-service-image-title"><a class="black-link" href="/CRISPR-Stable-Knockin-Cell-Line-Generation.html">CRISPR Stable Knockin Cell Line Generation Service</a></div>
									</div>
								</div>
							</li>
						</ul>
					</div>
				</div>
			</div>
			<div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
				<div class="row">
					<div class="module">
						<ul class="htmlcontent-home">
							<li class="col-xs-6 col-sm-6 col-md-6">
								<div class="banners">
									<div class="serviceimages" style="text-align: left;">
										<a href="/CRISPR-Bacterial-Genome-Editing.html"> <img src="/pub/media/catalog/category/imgs/perfect_page/Related-Products-tube.png" /> <br /> </a>
										<div class="abm-custom-service-image-title"><a class="black-link" href="/CRISPR-Bacterial-Genome-Editing.html">CRISPR Bacterial Genome Editing Service</a></div>
									</div>
								</div>
							</li>
							<li class="col-xs-6 col-sm-6 col-md-6">
								<div class="banners">
									<div class="serviceimages" style="text-align: left;">
										<a href="/CRISPR-Gene-Deletion-CRISPR-miRNA-lncRNA-Knockout-Service.html"> <img src="/pub/media/catalog/category/imgs/perfect_page/Related-Products-tube.png" /> <br /> </a>
										<div class="abm-custom-service-image-title"><a class="black-link" href="/CRISPR-Gene-Deletion-CRISPR-miRNA-lncRNA-Knockout-Service.html">CRISPR Gene Deletion and miRNA/lncRNA Knockout Service</a></div>
									</div>
								</div>
							</li>
						</ul>
					</div>
				</div>
			</div>
		</div>
		<div class="col-lg-4"></div>
	</div>
</div>
<div id="abm-category-section5" class="abm-container-fluid">
	<div class="row">
		<div class="col-lg-8">
			<h3 class="abm-perfect-h3">Documents</h3>
			<div>
				<ul class="cleanlist">
					<li><img src="/pub/media/catalog/category/imgs/perfect_page/PDF.png" /> <a class="black-link doclinks" href="/pub/media/catalog/customservice/crispr-stable-knockout-cell-line/Revised - C208 CRISPR Stable KO Service Questionnaire-v2-3-1.pdf">CRISPR Stable KO Service Form</a></li>
				</ul>
			</div>
		</div>
		<div class="col-lg-4"></div>
	</div>
</div>
<div id="abm-category-section6" class="abm-container-fluid">
	<div class="row">
		<div class="col-lg-8">
			<h3 class="abm-perfect-h3">FAQs</h3>
			<table>
				<tbody>
					<tr>
						<td style="padding: 8px 0; width: 37px;"><a href="#collapseOne" data-toggle="collapse" data-parent="#accordion"> <i class="fa fa-angle-down OrderAngleDown"></i> </a></td>
						<td><a href="#collapseOne" data-toggle="collapse" data-parent="#accordion"><b class="customfaq black-link">What should customers provide for this service?</b></a></td>
					</tr>
				</tbody>
			</table>
			<table>
				<tbody>
					<tr>
						<td>
							<div id="collapseOne" class="panel-collapse collapse">
								<div class="abm-perfect-faqs-text">Please supply the cells for this service. For frozen cells, please ship at least 2 vials (at least 10^6 each) on dry ice. An alternative method is to send us two T25 flask of live cells per sample, at 90% confluency. The flasks should be filled with complete medium without any air bubbles and at room temperature. Please ensure that the sterilization procedure is strictly observed. To avoid delays over the weekend, we recommend shipping the cells on a Monday or a Tuesday. Frozen cells are preferred over live cells. If your cell line already contains an antibiotic resistance marker, please specify. Our vectors have puromycin resistance by default. If your cells require medium other than DMEM and RPMI, we will also require 1L of the specified medium and any applicable growth factors or supplements to be provided by you for the project. Please submit all components of the complete media (e.g. if growth factors, cytokines etc. are applicable) individually to eliminate potential degradation of components in the media during transit. Kindly include instructions for making the complete media in your shipment. Additionally, please provide at least 5 x coated 6-well plates and 10 x T25 flasks if the cells require specially treated culture vessels. Once you have shipped your cells, please forward us the tracking number for custom clearance. Information on how to ship cell samples to abm can be found on our support page: https://www.abmgood.com/Technical-Support.html Please place an order first prior to submitting your samples. All samples received must have the order confirmation number indicated. Any samples received without this piece of information will be disposed off immediately upon receipt to ensure that all customer information is held in strict confidence.</div>
							</div>
						</td>
					</tr>
				</tbody>
			</table>
			<table>
				<tbody>
					<tr>
						<td style="padding: 8px 0; width: 37px;"><a href="#collapseTwo" data-toggle="collapse" data-parent="#accordion"> <i class="fa fa-angle-down OrderAngleDown"></i> </a></td>
						<td><a href="#collapseTwo" data-toggle="collapse" data-parent="#accordion"><b class="customfaq black-link">Can this service be performed for canine derived cells?</b></a></td>
					</tr>
				</tbody>
			</table>
			<table>
				<tbody>
					<tr>
						<td>
							<div id="collapseTwo" class="panel-collapse collapse">
								<div class="abm-perfect-faqs-text">Yes, we can use sgRNA targeting canine genome to make stable KO cell lines. Please provide the specific cell medium for canine cell culture.</div>
							</div>
						</td>
					</tr>
				</tbody>
			</table>
			<table>
				<tbody>
					<tr>
						<td style="padding: 8px 0; width: 37px;"><a href="#collapseThree" data-toggle="collapse" data-parent="#accordion"> <i class="fa fa-angle-down OrderAngleDown"></i> </a></td>
						<td><a href="#collapseThree" data-toggle="collapse" data-parent="#accordion"><b class="customfaq black-link">Will CRISPR keep cutting the chromosome after the gene is edited?</b></a></td>
					</tr>
				</tbody>
			</table>
			<table>
				<tbody>
					<tr>
						<td>
							<div id="collapseThree" class="panel-collapse collapse">
								<div class="abm-perfect-faqs-text">CRISPR is sensitive to mismatches, so it is unlikely the CRISPR will keep cutting the chromosome after the gene is edited.</div>
							</div>
						</td>
					</tr>
				</tbody>
			</table>
			<table>
				<tbody>
					<tr>
						<td style="padding: 8px 0; width: 37px;"><a href="#collapseFour" data-toggle="collapse" data-parent="#accordion"> <i class="fa fa-angle-down OrderAngleDown"></i> </a></td>
						<td><a href="#collapseFour" data-toggle="collapse" data-parent="#accordion"><b class="customfaq black-link">Is Nuclease or Nickase used? Can you use the double nickase approach?</b></a></td>
					</tr>
				</tbody>
			</table>
			<table>
				<tbody>
					<tr>
						<td>
							<div id="collapseFour" class="panel-collapse collapse">
								<div class="abm-perfect-faqs-text">The approach used for the above service is sgRNA with Cas9 Nuclease, relying on NHEJ repair without a repair template. The Nickase approach is available but this will be at an additional charge and will be subject to a custom quotation as a different strategy will need to be devised.</div>
							</div>
						</td>
					</tr>
				</tbody>
			</table>
			<table>
				<tbody>
					<tr>
						<td style="padding: 8px 0; width: 37px;"><a href="#collapseFive" data-toggle="collapse" data-parent="#accordion"> <i class="fa fa-angle-down OrderAngleDown"></i> </a></td>
						<td><a href="#collapseFive" data-toggle="collapse" data-parent="#accordion"><b class="customfaq black-link">How can you detect a mutation if there is no selection (by PCR using primers flanking the modified site or the T7 endonuclease I assay)?</b></a></td>
					</tr>
				</tbody>
			</table>
			<table>
				<tbody>
					<tr>
						<td>
							<div id="collapseFive" class="panel-collapse collapse">
								<div class="abm-perfect-faqs-text">Yes, that is one method used. Other methods include Sanger sequencing, and Western Blot if the antibody is provided by the customer.</div>
							</div>
						</td>
					</tr>
				</tbody>
			</table>
			<table>
				<tbody>
					<tr>
						<td style="padding: 8px 0; width: 37px;"><a href="#collapseSix" data-toggle="collapse" data-parent="#accordion"> <i class="fa fa-angle-down OrderAngleDown"></i> </a></td>
						<td><a href="#collapseSix" data-toggle="collapse" data-parent="#accordion"><b class="customfaq black-link">How many sgRNAs are designed and synthesized for this service? Are pooled sgRNAs used?</b></a></td>
					</tr>
				</tbody>
			</table>
			<table>
				<tbody>
					<tr>
						<td>
							<div id="collapseSix" class="panel-collapse collapse">
								<div class="abm-perfect-faqs-text">We design 3 sgRNAs at the time of order placement, however only 1 sgRNA is used to achieve knockout. In the case that the first sgRNA does not allow for success, we would try another from the set of 3 synthesized sgRNAs.</div>
							</div>
						</td>
					</tr>
				</tbody>
			</table>
			<table>
				<tbody>
					<tr>
						<td style="padding: 8px 0; width: 37px;"><a href="#collapseSeven" data-toggle="collapse" data-parent="#accordion"> <i class="fa fa-angle-down OrderAngleDown"></i> </a></td>
						<td><a href="#collapseSeven" data-toggle="collapse" data-parent="#accordion"><b class="customfaq black-link">Can you offer off-target site prediction service and off-target site analysis service (Sanger sequence)?</b></a></td>
					</tr>
				</tbody>
			</table>
			<table>
				<tbody>
					<tr>
						<td>
							<div id="collapseSeven" class="panel-collapse collapse">
								<div class="abm-perfect-faqs-text">When we design the sgRNAs, the design software will predict the number of off-targets, which is usually 0 for the sgRNAs that we select. Off-target analysis: This will need to be done by NGS, whole genome sequencing or exome sequencing, which would require comparing the control (before) sample to the genome edited sample (after). If this is a desired option, we can provide a quote for it, but it will likely be at least $2200 for analyzing 1-2 clones by Exome Seq, and higher for whole genome sequencing.</div>
							</div>
						</td>
					</tr>
				</tbody>
			</table>
			<table>
				<tbody>
					<tr>
						<td style="padding: 8px 0; width: 37px;"><a href="#collapseEight" data-toggle="collapse" data-parent="#accordion"> <i class="fa fa-angle-down OrderAngleDown"></i> </a></td>
						<td><a href="#collapseEight" data-toggle="collapse" data-parent="#accordion"><b class="customfaq black-link">How is KO confirmed?</b></a></td>
					</tr>
				</tbody>
			</table>
			<table>
				<tbody>
					<tr>
						<td>
							<div id="collapseEight" class="panel-collapse collapse">
								<div class="abm-perfect-faqs-text">We perform Sanger sequencing of the locus of interest both to determine the nature of the INDEL in the edited genes to guarantee frameshift silencing, and to confirm that the genome is edited. For even greater certainty, we also offer Next Generation Sequencing confirmation (AI00100), which can detect WT alleles down to 10% of total. Being much more sensitive, NGS screening is ideal for detecting complete editing of multiple copy genes, or polyploid cell lines.</div>
							</div>
						</td>
					</tr>
				</tbody>
			</table>
		</div>
		<div class="col-lg-4"></div>
	</div>
</div>
<div id="abm-category-section7" class="abm-container-fluid">
	<div class="row">
		<div class="col-lg-8">
			<h3 class="abm-perfect-h3">Citations</h3>
			<div>
				<table>
					<tbody>
						<tr>
							<td><span class="citations-num"><a class="orange-link" href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1038/ncomms9371">01</a></span></td>
							<td>Kang, Y. J. et al. "Regulation of NKT cell-mediated immune responses to tumours and liver inflammation by mitochondrial PGAM5-Drp1 signalling." Nat. Commun. (2015) 6:8371 <a class="orange-link" href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1038/ncomms9371">doi: 10.1038/ncomms9371</a></td>
						</tr>
						<tr>
							<td><span class="citations-num"><a class="orange-link" href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1158/1940-6207.CAPR-15-0338">02</a></span></td>
							<td>Jiang, G. et al. "Isorhapontigenin (ISO) inhibits invasive bladder cancer (BC) formation in vivo and human BC invasion in vitro by targeting STAT1/FOXO1 Axis." Cancer Prev Res. Published Online First April 14, 2016.<a class="orange-link" href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1158/1940-6207.CAPR-15-0338">doi: 10.1158/1940-6207.CAPR-15-0338</a></td>
						</tr>
						<tr>
							<td width="5%"><span class="citations-num"><a class="orange-link" href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1136/gutjnl-2015-309359">03</a></span></td>
							<td width="95%">Okugawa, Y. et al. "Clinical significance of SNORA42 as an oncogene and a prognostic biomarker in colorectal cancer." Gut (2015)<a class="orange-link" href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1136/gutjnl-2015-309359">doi:10.1136/gutjnl-2015-309359</a></td>
						</tr>
					</tbody>
				</table>
			</div>
		</div>
		<div class="col-lg-4"></div>
	</div>
</div>